rgbd/include/rgbd/Image.h

```diff
@@ -40,29 +41,31 @@ enum DepthStorageType
 
 class Image {
 
-    friend class Server;
-    friend class ServerShmOnly;
-    friend class Client;
-    friend class SharedMemClient;
+    friend class ServerRGBD;
```

Comment:
Do we need these? Is it safe? And although it didn't change: 'View' seems most 'inappropriate'

Reply:
`View` and `ClientRGBD` are removed. `View` did use the access, but doesn't need it. `ClientRGBD` didn't use the access, so was also removed. The other classes use it so they can read/write more efficiently.